app/Http/Requests/StoreNewRegistrationRequest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Http\Request;
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreNewRegistrationRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        // Todo: replace with check that user cna make this request rather than wave them through.
+        return true;
```

Comment:
Obviously - this needs doing... but maybe nit in this PR.

Reply:
yep, or possibly even in alpha, even if it make me twitch to leave it out.